### ISSUE TITLE
Editing a jdbc connection in 1.5.1 throws internal server error

### ISSUE THREAD

**Issue by jomar83** [apache/airflow#395]
Editing an existing jdbc connection results in:

```
File "/usr/local/lib/python2.7/site-packages/airflow-1.5.1-py2.7.egg/airflow/www/app.py", line 1765, in on_form_prefill
    field = getattr(form, field)
AttributeError: 'ConnectionForm' object has no attribute 'extra__jdbc__drv_path'
```

At the same time creating a new jdbc connection doesn't show the extra form fields anymore.

**Comment by jomar83** [comment]
If form_columns is used for using only specific fields from the model or for forcing a certain order of fields, form_extra_fields need to be in there as well, otherwise they will be ignored. 

Fixed by #396

**Comment by jomar83** [comment]
Premature close, will close when fix is merged.
